oidc-api/src/main/java/uk/gov/di/authentication/oidc/exceptions/AccountInterventionException.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.di.authentication.oidc.exceptions;
+
+public class AccountInterventionException extends RuntimeException {
+    public AccountInterventionException(Exception cause) {
```

Comment:
Looking at other custom exceptions in this dir, the general pattern is 
`public MyException(String message) {
        super(message);
    }`
    and a message is passed when the exception is called. Is there a reason to break that pattern here?

Reply:
I'll can add a message, though I think the context is all the cause. Difficult to really know if it's helpful or just noise outside the context of looking at bugs and how that is handled by the calling code.